package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "patternfly-ng",
-  "version": "0.0.5",
+  "version": "0.0.0-semantic-release",
```

Comment:
For semantic release - it needs to be set by the build system and library so we pick a string identifier instead of a number.